src/components/MyHeader.jsx

```diff
@@ -0,0 +1,120 @@
+import React, { Component } from "react";
+import "bootstrap/dist/css/bootstrap.css";
+import TodoList from "./TodoList";
+import UUID from "uuid";
+
+class MyHeader extends Component {
+  state = {
+    tasks: [],
+    nextId: UUID(),
+    input: "",
+    editItem: false
+  };
+  handleInputChange = e => {
+    this.setState({
+      ...this.state,
+      input: e.target.value
+    });
+  };
+
+  addTodo = e => {
+    e.preventDefault();
+    /**
+     * type task = {
+     *  id: UUID,
+     *  content: String,
+     *  completed: bool,
+     * }
+     */
+
+    if (this.state.input === "") {
+      alert("todo cannot be empty");
+    } else {
+      const addTask = {
+        id: this.state.nextId,
+        content: this.state.input
+      };
+
+      const updatedTasks = [...this.state.tasks, addTask];
+
+      this.setState(currentState => {
+        return {
+          ...currentState,
+          tasks: updatedTasks,
+          nextId: UUID(),
+          input: ""
+        };
+      });
+    }
+  };
+
+  nullWarning() {
+    let addWarning = "btn m-2 btn-";
+    addWarning += this.state.input === "" ? "warning" : "success";
+    return addWarning;
+  }
```

Comment:
it's a good idea to put line between your functions. it's much cleaner.

Reply:
@theAndrewCline sure will follow that.